packages/forms/src/lib/mask/README.md

```diff
@@ -0,0 +1,43 @@
+# @acpaas-ui/ngx-components/forms
+
+With the @acpaas-ui mask module you can mask input fields.
```

Comment:
... you can make input fields that only accept a specific format.

Het is een iets te circulaire omschrijving anders.